index.html

```diff
@@ -45,6 +45,342 @@ <h2 class="supporting">
                 </ul>
             </span>
         </section>
+        <section class="recent-works">
```

Comment:
- [OPTIONAL] Well done implementing the requested changes by the previous reviewers. however, there is an issue with your design that I would highly recommend you to solve.
  - One of the core requirements for this milestone is to be responsive from `375px` to the lowest desktop width which is `768px`. what I can see right now is that you have given exact specific width to your `body` which has made it unresponsive. so kindly consider working on that.
  -  Minimum mobile screen size should be `375`px. Errors in screen sizes less than `375`px should be ignored. Otherwise, mobile version should be responsive until the desktop screen size.